internal/service/studies/main.go

```diff
@@ -17,6 +21,83 @@ func New() *Service {
 	}
 }
 
+func (s *Service) CreateStudy(userID uuid.UUID, studyData openapi.StudyCreateRequest) (*types.Study, error) {
+	if strings.TrimSpace(studyData.Title) == "" {
+		return nil, errors.New("study title is required")
```

Comment:
I think we should come up with some regex for a valid study name and enforce it on the frontend and backend. e.g `^\w[\w\s\-]{2,48}\w$`